scipy/linalg/src/_batched_linalg.h

```diff
@@ -0,0 +1,305 @@
+#pragma once
+
+#include "Python.h"
+#include <limits>
+#include <cassert>
+#include "numpy/arrayobject.h"
+#include "npy_cblas.h"
+#include "_lapack_trampolines.h"
+#include "_npymath.h"
+
+
+using namespace _numpymath;
```

Comment:
why is this namespace `_numpymath` required? Can't we not template using the regular dependencies to NumPy without our own extra boilerplates?